cli/cmd/generate.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	generator "github.com/opencontrol/oscalkit/generator"
+	"github.com/opencontrol/oscalkit/templates"
+	"github.com/opencontrol/oscalkit/types/oscal/catalog"
+	"github.com/urfave/cli"
+)
+
+var profilePath string
+
+//Generate Cli command to generate go code for controls
+var Generate = cli.Command{
+	Name:  "generate",
+	Usage: "generates go code against provided catalogs and profile",
```

Comment:
for consistency, since it doesn't yet generate code from catalogs

```suggestion
	Usage: "generates go code against provided profile",
```

Reply:
Done